scopes/defender/tester/test.cmd.tsx

```diff
@@ -21,8 +21,15 @@ type TestFlags = {
 };
 
 export class TestCmd implements Command {
-  name = 'test [pattern]';
-  description = 'test set of components in your workspace';
+  name = 'test [component-pattern]';
+  description = 'Tests components in your workspace';
+  arguments = [
+    {
+      name: 'component-pattern',
+      description:
```

Comment:
(yes, it's very confusing and must be fixed) this "pattern" is the old pattern, which doesn't have the ability of using "!".  (that's why it didn't have the constant pattern variable as other commands).

Reply:
fixed